events/0301_rb.html

```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+
+    <title>A Day In The Life : Developers | Girl Develop It Ann Arbor</title>
+
+    <meta name="description" content="">
+    <meta name="author" content="Girl Develop It">
+
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+    <link rel="stylesheet" href="css/reveal.css">
+    <link rel="stylesheet" href="css/theme/gdiaa.css" id="theme">
+    <link rel="stylesheet" href="lib/css/rainbow.css">
+    <link rel="stylesheet" href="css/print/pdf.css" type="text/css" media="print">
+
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+
+    <script src="lib/js/head.min.js"></script>
+
+    <!--[if lt IE 9]>
+    <script src="lib/js/html5shiv.js"></script>
+    <![endif]-->
+</head>
+
+<body>
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+
+            <!-- Event Intro -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="450px" />
+                <div class="box--small">
+                    <h3 class="event__title">Intro to Ruby<br /><small><span class="green">4 Session Workshop</span></small></h3>
+                    <p><small>@gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #IntroToRuby</small></p>
```

Comment:
@krismyth let's add Andrew Sardone and his @andrewa2 twitter name to the beginning of the line - here and in the closing slide